File: androidApp/src/main/java/com/commuto/interfacemobile/android/ui/CurrentTab.kt
package com.commuto.interfacemobile.android.ui

import com.commuto.interfacemobile.android.ui.offer.OffersComposable
import com.commuto.interfacemobile.android.ui.settlement.SettlementMethodsComposable
import com.commuto.interfacemobile.android.ui.swap.SwapsComposable

/**
 * Indicates which tab view should be displayed to the user.
 *
 * @property OFFERS Indicates that [OffersComposable] should be displayed to the user.
 * @property SWAPS Indicates that [SwapsComposable] should be displayed to the user.
 * @property SETTLEMENT_METHODS Indicates that [SettlementMethodsComposable] should be displayed to the user.
 */
enum class CurrentTab {
    OFFERS,
    SWAPS,
    SETTLEMENT_METHODS;
}

File: iosApp/iosApp/ui/CurrentTab.swift
//
//  CurrentTab.swift
//  iosApp
//
//  Created by jimmyt on 8/9/22.
//  Copyright © 2022 orgName. All rights reserved.
//

/**
 Indicates which tab view should be displayed to the user.
 */
enum CurrentTab {
    /**
     Indicates that `OffersView` should be displayed to the user.
     */
    case offers
    /**
     Indicates that `SwapsView` should be displayed to the user.
     */
    case swaps
    /**
     Indicates that `SettlementMethodsView` should be displayed to the user.
     */
    case settlementMethods
}
